docs/guide/docs/build/channels.md

```diff
@@ -3,17 +3,15 @@ id: channels
 title: Messaging Channels
 ---
 
```

Comment:
I would leave the documentation of channels mostly generic here. I think the specific details of setting up individual channels should be in their respective README.md, otherwise this page will be quickly bloated.

We should just show a list of channels, their supported features and a link for more details

Reply:
Problem is, each channel has there own configuration. Its also nice to have a guide on own to setup a bot for a specific channel.